### PR TITLE
force rebase to origin on circleci

### PR DESCRIPTION
**Pull request by johnnie502**
This is for https://github.com/Medology/stdcheck.com/issues/2039

### CODE REVIEW

**Comment by johnnie502** [comment]
@TaysirTayyab something wrong with this repo, can you take a look? https://circleci.com/gh/Medology/phpunit-parallel-runner/35?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: circle.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ dependencies:
     - "vendor/"
 
   override:
+    - git config --global user.email "ci@medology.com" && git config --global user.name "Circle CI" && git rebase origin
```
Comment (TaysirTayyab) [inline circle.yml]: The problem in the build is that we removed the repo from code climate. You'll need to do 2 things to fix:
- remove 19 in this file (`./circleci/bin/php vendor/bin/test-reporter`)
- remove the code climate dev dependency in the `composer.json` file

**Comment by medology-bugman** [comment]
Paging @jose-ochoa to review the back-end work in this pull request!

I couldn't recognize who these files were relevant to:
circle.yml
composer.json

**Comment by jose-ochoa** [comment]
!reassign back-end @TaysirTayyab 
As I don't possess the proper ninja skillz for this at this time.

**Comment by medology-bugman** [comment]
Paging @TaysirTayyab to review the back-end work in this pull request!

**Pull request merged**